src/reducers/commentReducers/comment.js

```diff
@@ -0,0 +1,90 @@
+import * as actionType from '../../actions/commentAction/commentType';
+import updateObject from '../../helpers/store/utility';
+import commentAuthor from '../../helpers/utilities/addCommentAuthor';
+
+
+const initialState = {
+  isLoading: false,
+  response: null,
+  error: null,
+  success: null,
+  postIsLoading: false,
+  getSuccess: null,
+};
+
+const getCommentStartState = {
+  isLoading: true,
+  error: false,
+};
+
+const getCommentSuccessState = (state, action) => updateObject(state, {
+  isLoading: false,
+  response: action.payload,
+  success: true,
+  postIsLoading: false,
+  getSuccess: true,
+});
+
+const getCommentFailedState = (state, action) => updateObject(state, {
+  isLoading: false,
+  error: true,
+  response: action.payload,
+  success: false,
+  postIsLoading: false,
+  getSuccess: false,
+});
+
+const postCommentStartState = {
+  postIsLoading: true,
+  error: false,
+};
+
+
+const postCommentFailedState = (state, action) => updateObject(state, {
+  isLoading: false,
+  error: true,
+  success: false,
+  postResponse: action.payload,
+  postIsLoading: false,
+  getSuccess: true,
+});
+
+const postCommentSuccessState = (state, action) => updateObject(state, {
+  isLoading: false,
+  response: [
+    commentAuthor(action.payload),
+    ...(state.response ? [...state.response] : []),
+  ],
+  success: true,
+  postIsLoading: false,
+  getSuccess: true,
+});
+
+
+const commentReducers = (state = initialState, action) => {
+  switch (action.type) {
+    case actionType.GET_COMMENT_START:
+      return updateObject(initialState, getCommentStartState);
```

Comment:
Hi @idmega2000. You should pass the state and not the `initialState`.

Reply:
Thank you. It has been fixed.